.travis/coveralls_uploader.js

```diff
@@ -0,0 +1,44 @@
+var fs = require('fs');
+var request = require('request');
+const md5File = require('md5-file')
+
+var repo_token = process.env.COVERALLS_REPO_TOKEN;
+process.stdout.write("token");
+process.stdout.write(process.env.COVERALLS_REPO_TOKEN);
+
+if (process.env.COVERALLS_REPO_TOKEN) {
```

Comment:
I think it would be best to externalize the environment variables dependencies.

This way the script is not depending on TRAVIS - we should be able to upload from any CI.
See how .coveralls.yml file is used in coveralls for ruby. 
It is the place to define CI-Coveralls mappings.
And define coveralls variables from CI-specific variables.